e2e-tests/support/trade-finance-manager/login.js

```diff
@@ -0,0 +1,55 @@
+import { issueValidJWT } from '../utils/crypto';
+
+const crypto = require('crypto');
+const { TFM_URL } = require('../../e2e-fixtures/constants.fixture');
+
+const LOGIN_SESSION_MAX_AGE = 1800; // 30 min
+
+/**
+ * login
+ * Ensure a user is logged in to TFM.
+ * Return JWT token for API use.
+ * 1) Create season id
+ * 2) Set session id in MondoDB
+ * 3) Set session in Redis
+ * 4) Set session id in Cookie, if isSessionForAPICall is false (default behaviour)
+ * 5) Redirects to TFM root /, if isSessionForAPICall is false (default behaviour)
+ * @param {Object} user: User object for login
+ * @param {Boolean} isSessionForAPICall: if true no cookie set and no redirect will happen.
+ * @returns {Promise<String>} TFM JWT token.
+ *
+ * @example
+ * // Returns "Bearer ..."
+ * cy.login({user});
+ *
+ * @example
+ * // Returns "Bearer ..."
+ * cy.login({user, isSessionForAPICall: true}).then((token) => apiCall(token));
+ *
+ */
+const login = (user, isSessionForAPICall = false) => {
```

Comment:
As this login is now a cypress command, we do not get any intelisense as to what the parameters are, leaving us forced to declare a `isSessionForApiCall` prior to using it -- We should change this to an object param I think

Reply:
Updated this to reflect docs